### DISCUSSION TITLE
Not able to reapply

### DISCUSSION THREAD

**Discussion by Ayush-Kumar-web** [community/community#77392] [GitHub Education]
### Hi everyone,

Hey Everyone,
I applied for github student developer pack
and everything went good.

![image](https://github.com/community/community/assets/69073878/1b7a2c44-7527-4014-ab35-ef5c4d24fea7)

but i immediately got an email cancelling my pack request saying I should use my device camera to upload photograph rather than uploading.

But I can't seem to reapply for pack again from that same email though the previous one got cancelled 

![image](https://github.com/community/community/assets/69073878/2a907fb1-77f0-4a7a-ada9-a394f4ed3f21)

Please help

**Comment by CheetahCodes21** [comment]
It seems like there might be a slight discrepancy in the verification process for your GitHub Student Developer Pack application. The cancellation email you received indicates that you need to provide a photo of your student ID using your device camera rather than uploading an image file. This is likely an attempt to enhance the verification process and ensure that only eligible students receive the pack.

However, the inability to reapply from the same email after the cancellation is concerning. To resolve this issue, I recommend contacting GitHub Education support directly. They can assist you in understanding the specific requirements for reapplying and guide you through the process.

Here's how to reach GitHub Education support:

1. **Visit the GitHub Education support page:** [https://education.github.com/](https://education.github.com/)

2. **Click on the "Contact us" button:** This will open a contact form.

3. **Select the appropriate category:** Choose "Billing & Plans" and then "GitHub Student Developer Pack" as the subcategory.

4. **Provide a clear and concise explanation of your situation:** Briefly explain that your application was canceled due to the photo requirement and that you're unable to reapply using the same email.

5. **Attach any relevant screenshots or documentation:** If you have screenshots of the cancellation email or any other supporting documents, include them in the email.

6. **Submit the form and wait for a response:** The GitHub Education support team will review your request and provide you with further instructions or assistance.

By contacting GitHub Education support directly, you can ensure that your situation is properly addressed and that you receive the necessary guidance to proceed with your application.
